.github/workflows/ci.yml

```
@@ -0,0 +1,119 @@
+        run: npm audit --audit-level=high
+        continue-on-error: true
+
+  pr-summary:
+    name: PR Summary
+    runs-on: ubuntu-latest
+    needs: [lint-and-typecheck, build, security-audit]
+    if: always()
+    
+    steps:
+      - name: Check job results
+        run: |
```

actionlint (1.7.7)

"github.head_ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/security-for-github-actions/security-guides/security-hardening-for-github-actions for more details

(expression)